tests/pass-dep/libc/libc-pipe.rs

```diff
@@ -110,3 +112,10 @@ fn test_pipe_array() {
     let mut fds: [i32; 2] = [0; 2];
     assert_eq!(unsafe { pipe(&mut fds) }, 0);
 }
+
+// Test if O_NONBLOCK flag is supported.
+fn test_flag() {
```

Comment:
Maybe call this `test_pipe2`? It seems so far we don't have any test for that *oops*.

The `cfg` needs to be repeated here.